container_pipeline/model_tmp/containers.py

```diff
@@ -0,0 +1,81 @@
+import subprocess
+
+#from django.db import models
+#from django.conf import settings
+from os import path
+from json import load, dump
+
+# TODO : Change implementation into proper model and move this class into model package.
+
+
+def form_dockerfile_link(git_url, git_path, git_branch, target_file):
+    """
+    Helper function to generate docker file link.
+    :param git_url: The url of the git repository.
+    :param git_path: The path, relative to the git repository root, where the file resides.
+    :param git_branch: The repository branch where the file resides.
+    :param target_file: The name of the target file.
+    :return: The reachable link to the dockerfile.
+    """
+
+    # TODO : Move this into a lib.
+
+    link_url = None
+    if "github" in git_url or "gitlab" in git_url:
+        link_url = str.format(
+            "{git_url}/blob/{git_branch}/{git_path}/{target_file}",
+            git_url=git_url,
+            git_branch=git_branch,
+            git_path=git_path,
+            target_file=target_file
+        )
+    return link_url
+
+
+class ContainerLinksModel(object):
+    """Represents information about the container, stored in the model."""
+
+    def __init__(self, data_dir="/srv/pipeline-logs"):
```

Comment:
define this at the top of (this) file like
```
DATA_FILE_DIR_DEFAULT = "/srv/pipeline-logs"
```
and then use the constant (written in caps) as the default value for parameter of the function.